catalog-entities/marketplace/plugins/quickstart.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: extensions.backstage.io/v1alpha1
+kind: Plugin
+metadata:
+  name: quickstart
+  namespace: rhdh
+  title: Quickstart
+  description: "Guided quickstarts to help users discover and set up key features in Red Hat Developer Hub."
+  annotations:
+    extensions.backstage.io/verified-by: Red Hat
```

Not verified according to [this](https://docs.google.com/spreadsheets/d/1K_LqKYma9nRM5teKD-uCnbB3qKC8Hk2E5jvcpMSfjTs/edit?gid=771893283#gid=771893283).

```suggestion
    extensions.backstage.io/pre-installed: 'true'
```